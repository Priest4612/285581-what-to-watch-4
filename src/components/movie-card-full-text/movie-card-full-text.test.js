import React from 'react';
import renderer from 'react-test-renderer';

import filmsMock from '../../mocks/test-films-mock.json';

import MovieCardFullText from './movie-card-full-text.jsx';


it(`Render MovieCardFullText`, () => {
  const {director, starring, description} = filmsMock[2];
  const tree = renderer
    .create(<MovieCardFullText
      director={director}
      starring={starring}
      description={description}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
