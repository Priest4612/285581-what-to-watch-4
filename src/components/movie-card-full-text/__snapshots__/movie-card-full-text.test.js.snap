// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCardFullText 1`] = `
<div
  className="movie-card__text"
>
  <p
    className="movie-card__director"
  >
    <strong>
      Director: Слёзы, одна за другой, покатились по морщинистым щекам деда. Он мотал головой и плакал, просто молча плакал…
    </strong>
  </p>
  <p
    className="movie-card__starring"
  >
    <strong>
      Starring: Стивен Спилберг, Питер Джексон
    </strong>
  </p>
</div>
`;
